src/styles/Blocks/card.css

```
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  padding: 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 16px 3px 0;
```

```suggestion
  margin: 0 auto;
```

not `3px`, card may become wider 😉